README.rst

```diff
@@ -191,49 +189,36 @@ meta taking priority.
             #     'download_slot': 'quotes.toscrape.com'
             # }
 
-There is a shortcut, in case a request uses the same parameters as
-defined in the ``ZYTE_API_DEFAULT_PARAMS`` setting, without any further
-customization - the ``zyte_api`` meta key can be set to ``True`` or ``{}``:
+``ZYTE_API_DEFAULT_PARAMS`` does not make requests automatically go through
+Zyte Data API. See :ref:`enabled`.
 
-.. code-block:: python
+Parameters in ``ZYTE_API_DEFAULT_PARAMS`` are merged with parameters set via
+the ``zyte_api`` meta key, with the values in meta taking priority.
 
-    import scrapy
 
+.. _enabled:
 
-    class SampleQuotesSpider(scrapy.Spider):
-        name = "sample_quotes"
+Controlling which requests go through Zyte Data API
+---------------------------------------------------
 
-        custom_settings = {
-            "ZYTE_API_DEFAULT_PARAMS": {
-                "browserHtml": True,
-            }
-        }
+The ``ZYTE_API_ENABLED`` setting can be used to control whether all, none, or
+some requests go through Zyte Data API. It supports the following values:
 
-        def start_requests(self):
-            yield scrapy.Request(
-                url="http://quotes.toscrape.com/",
-                callback=self.parse,
-                meta={"zyte_api": True},
-            )
+-   ``None`` (default): only requests where the ``zyte_api`` key in
+    Request.meta_ is set to ``True`` or set to a dictionary go through Zyte
+    Data API.
```

Comment:
I'm not too sure about this part, since at first glance, the `ZYTE_API_ENABLED` setting seems to only need `True` or `False` with some boolean default set to it. 

What do you think about having the following?
- `ZYTE_API_ENABLED=False` turn off the plugin completely.
- `ZYTE_API_ENABLED=True` _(default)_ turn it on unless the `zyte_api` key in `Request.meta` is set explicitly set to `False`.

and then a separate setting to control the behavior of the defaults.